lib/acts_as_list/active_record/acts/position_column_method_definer.rb

```diff
@@ -35,11 +35,14 @@ def self.define_class_methods(caller_class, position_column, touch_on_update)
       end
 
       define_singleton_method :decrement_all do
-        update_all_with_touch "#{quoted_position_column} = (#{quoted_position_column_with_table_name} - 1)"
+        min = minimum(position_column)
```

Comment:
You're removing all usages of `update_all_with_touch` but not actually removing the method. We should remove the method if it's not needed.